README.md

```diff
@@ -541,17 +573,18 @@ cd explore-education-statistics\src\GovUk.Education.ExploreEducationStatistics.A
 dotnet ef migrations add EES1234_MigrationNameGoesHere --context UsersAndRolesDbContext --output-dir Migrations/UsersAndRolesMigrations -v
 ```
 
-### Resetting the storage emulator
+### Resetting Azurite
 
-During development you might want to reset your storage emulator to clear out all data from 
+During development you might want to reset your Azurite instance to clear out all data from 
 blobs, queues and tables. This is typically done at the same time as resetting the databases.
 
-To delete all data in the storage emulator:
+To delete all data in Azurite simply delete the azurite docker container and recreate it:
```

Comment:
I think you need to remove the Azurite volume in docker too?

Also capitalise `azurite` here to be consistent.